docs/cvl/methods.md

```diff
@@ -759,6 +755,45 @@ The behavior of the `AUTO` summary depends on the type of call[^opcodes]:
 This summary is a short syntax for a summary that contains an `assert false;` and checks that the summarized method is not reached.
 This can be useful for instance, in the presence of unresolved calls in combination with the `unresolved external` syntax to ensure that every unresolved call is actually dispatched correctly (i.e. use `unresolved external in _._ => DISPATCH [...] default ASSERT_FALSE`). It also enables more optimizations in the Prover and may lead to shorter running times.
 
+#### Practical patterns and tips
+
+- Contract aliases with `using` help target specific receivers and keep specs readable. For example, a multi-contract scene can alias tokens, pools, and controllers once and reuse them throughout the spec:
+
+  ```cvl
+  using StataTokenV2Harness as _StaticATokenLM;
+  using SymbolicLendingPool as _SymbolicLendingPool;
+  using RewardsControllerHarness as _RewardsController;
+  using ATokenInstance as _AToken;
+  ```
+
+- Prefer `envfree` for pure/view‑like calls in specs to avoid unnecessary environment coupling:
```

Comment:
it's good to put envfree for pure/view like functions, but it's not always correct. Second, what is environment coupling? we're making up terminology. 

You put envfree if you're absolutely sure you do not depend on msg.sender, block.* variables and others. This makes specs easier to write and less cluttered. But I don't understand why we need a separate section for tips & tricks instead of putting the tip where envfreee is defined and explained